packages/excalidraw/components/App.tsx

```diff
@@ -7900,6 +7901,31 @@ class App extends React.Component<AppProps, AppState> {
           !this.state.editingTextElement &&
           this.state.activeEmbeddable?.state !== "active"
         ) {
+
+          if(!this.state.editingLinearElement && !this.state.editingFrame && !this.state.resizingElement){
+
+            // this determines the threshold for dragging the element initially
+            const MOVEMENT_BUFFER = 4;
+  
+            // this calculates the buffer for the movement based on the zoom level
+            const ZOOM_RELETIVE_MOVEMENT_BUFFER = MOVEMENT_BUFFER / this.state.zoom.value;
+            
+            if(typeof(ZOOM_RELETIVE_MOVEMENT_BUFFER) === 'number'){
+  
+              if (
+                pointDistance(
+                  pointFrom(pointerCoords.x, pointerCoords.y),
+                  pointFrom(pointerDownState.origin.x, pointerDownState.origin.y),
+                ) < ZOOM_RELETIVE_MOVEMENT_BUFFER && !elementStartedMoving
```

Comment:
the quick check for `!elementStartedMoving` should be first in the `if`, so that the distance calc. happens only when it isn't moving already

Reply:
yes this makes sense.